### DISCUSSION TITLE
th_to_csv_win64.exe does not convert contact forces correctly

### DISCUSSION THREAD

**Discussion by wmcnCG** [OpenRadioss/OpenRadioss#2451] [Q&A]
I am running a ball-plate impact simulation so I expect the normal force to start and end at 0.

I see the expected result when I plot the normal force components in HyperWorks after opening the T01 file:
![image](https://github.com/OpenRadioss/OpenRadioss/assets/109186482/7913ef10-d3b1-4262-9222-358044b83aa6)

However, here are the "same" normal force components plotted in Excel after I convert the t01 to csv format:
![image](https://github.com/OpenRadioss/OpenRadioss/assets/109186482/8a7b6ebe-c95b-4c11-a608-132b07fcd9b6)

**Comment by sebastienVilleneuve** [comment]
Hi @wmcnCG 

what you are seeing are impulse .
In order to have forces , you need to add /TH/TITLE card in the 1.rad file then rerun the engine .
( cf https://github.com/OpenRadioss/Tools/tree/main/output_converters/th_to_csv )

You will have a _TITLES file in same directory than the T01 , that is used by the th_to_csv converter .

Regards,
Sébastien

**Comment by wmcnCG** [comment]
Ah, great. Thank you for the quick response.

**Comment by sebastienVilleneuve** [comment]
You're welcome

**Comment by MarianBulla** [comment]
Hi,
yes, as mentioned by Sebastien, it is the impulse which is stored.
Please see this video: https://www.youtube.com/watch?v=EPZenaaDDak&t=298s
At time of 4:50 min. I'm explaining it more in detail.

Hope it helps you ?

Thanks and best,
Marian

**Comment by wmcnCG** [comment]
@sebastienVilleneuve @MarianBulla 
Adding /TH/TITLE to the engine file generated the _TITLES file and added the names to the output columns in the CSV (e.g., FNX, FNY, etc.). However, it did not convert the impulses to forces. Do you have a demo simulation where the forces appear correctly in the CSV file? Thanks.

**Comment by sebastienVilleneuve** [comment]
Hi @wmcnCG 

Can you share the model ?

Thanks,
Sébastien

**Comment by wmcnCG** [comment]
I cannot share it, sorry. I will see if I can recreate the issue using the BirdStrike example.

**Comment by elequiniou** [comment]
Hi, I propose to convert to discussion, waiting for additional feedback, thanks!

**Comment by rzadki22** [comment]
Hi, I got similar problem. See description in the last post in this thread: https://github.com/orgs/OpenRadioss/discussions/867

Beat regards,
Filip
